thermocyclerhelper/libraries/thermocyclers/source.rb

```diff
@@ -120,20 +107,16 @@ def export_measurements(thermocycler:)
 
   private
 
-  # Test whether an array of items is a single 96 well plate
-  #
-  # @param items [Array<Item>]
-  # @return [Boolean]
-  def single_96well_plate?(items)
-    items.length == 1 && is_96well_plate?(items.first)
-  end
-
   # Test whether an item is a 96 well plate
   #
   # @param item [Item]
   # @return [Boolean]
-  def is_96well_plate?(item)
-    item.collection? && item.capacity == 96
+  def is_plate?(item)
```

Comment:
I'd consider whether this method was meant to test whether it is a plate or whether it has 96 wells.

Reply:
Fair, I was trying to see if it was a plate.     And not a stripwell.   Was important to determine since a plate cannot be combined with other plates on a thermocycler.   Where a stripwell can.

This may not be something that should be handled here though.  It may be the duty of the protocol to know what type of items it is working with and if those items can be combined or not.